### PR TITLE
Anwser on filtering on mailbox

### PR DESCRIPTION
**Pull request by rouazana**

### CODE REVIEW

File: Sprint #19.adoc [hunk 1]
```diff
@@ -39,6 +39,7 @@
 * Do we want to manage quota on upload ? Ask that with INBOX and UX team
 ** Answer : We don't care right now
 * How hard would it be to filter on mailbox and/or users when use Filter on GetMessagesList
+** Answer: very. Off course no user indexing at all. James filtering API needs a Mailbox to search on. So it is not possible to compose any filter containing requests such as (inMailboxes=... or notInMailboxes=...). If we don't need compositing or a very simple compositing (2 clauses) the problem can be solved in JMAP backend. To have a plain support we need to end or work on unifying UID, ie a message should have a single id whatever is its mailbox. The good news: elasticsearch is already basing its indexing on such an UID, so maybe extending the API could fit our need for now.
```
Comment (mbaechler) [inline Sprint #19.adoc:42]: What do you have in mind about extending the API ?

Comment (rouazana) [inline Sprint #19.adoc:42]: Having a new API endpoint that doesn't need a Maibox to make a search query.

Comment (mbaechler) [inline Sprint #19.adoc:42]: And maybe adding a Criteria about mailbox filtering ? But then, what's the point of the previous API ?

Comment (rouazana) [inline Sprint #19.adoc:42]: Extending the Api (and maybe implementing capabilities for search backends) allows not to modify other existing implementations.
But as we have a few search implementations, that's maybe not a big deal.

**Comment by mbaechler** [comment]
:+1:

**Comment by aduprat** [comment]
:+1:

**Pull request merged**